docs/.vitepress/config.js

```diff
@@ -31,18 +112,25 @@ export default defineConfig({
   vue: {
     template: {
       compilerOptions: {
-        isCustomElement: (tag) => tag.includes('-example'),
+        isCustomElement: (tag) => /(-example|-color-table|example-container)$/.test(tag),
       },
     },
   },
   vite: {
     plugins: [
       uno({
-        presets: [presetWarp()],
+        presets: [presetWarp(), presetDocs()],
         mode: 'shadow-dom',
-        safelist: [...classes, ...docsClasses],
+        shortcuts: [{
+          'ex-font': 'pd-text-sm font-bold pd-font-mono pd-text-white',
+          'ex-font-dark': 'pd-text-sm font-bold pd-font-mono pd-text-slate-900',
+          'ex-box': 'ex-font p-24 rounded pd-shadow-xl flex items-center justify-center',
+          'ex-inner-box': 'p-24 rounded pd-shadow-xl mx-auto pd-bg-white pd-text-slate-500 max-w-[300]',
+          'ex-pic-no': 'absolute top-10 left-10 h-32 w-32 text-center pd-bg-white rounded-full pd-text-slate-800 leading-[32]',
+        }],
+        safelist: [...classes, ...supported, ...docsClasses],
```

Comment:
Would it make sense to rename those to something like `...componentClasses, ...supportedClasses` (or `...cssClasses`?). Just so it's easier to associate with respective part of the docs.